src/screens/gm/tweak.cpp

```diff
@@ -609,57 +627,6 @@ GuiShipTweakPlayer::GuiShipTweakPlayer(GuiContainer* owner)
     }
 }
 
-void GuiShipTweakPlayer::onDraw(sf::RenderTarget& window)
```

Comment:
Specific reason you moved this? The order with the 2nd player tweak class is now odd. And the diff is more complex.

Reply:
None, can undo